### ISSUE TITLE
npm run postinstall results in error

### ISSUE THREAD

**Issue by fallenturtle** [cloudsociety/plugin-node-scsstojson#1]
I'm using patternlab 3.0.0-alpha.16

I installed your plugin but when I run npm run postinstall I get an error. Here's the log:

0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node/10.3.0/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'postinstall' ]
2 info using npm@6.1.0
3 info using node@v10.3.0
4 verbose stack Error: missing script: postinstall
4 verbose stack     at run (/usr/local/lib/node_modules/npm/lib/run-script.js:151:19)
4 verbose stack     at /usr/local/lib/node_modules/npm/lib/run-script.js:61:5
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:115:5
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:418:5
4 verbose stack     at checkBinReferences_ (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:373:45)
4 verbose stack     at final (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:416:3)
4 verbose stack     at then (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:160:5)
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:280:12
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/graceful-fs/graceful-fs.js:78:16
4 verbose stack     at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:440:3)
5 verbose cwd /Users/abailey/Sites/jbs-corp-wp/wp-content/themes/jbscorp
6 verbose Darwin 16.7.0
7 verbose argv "/usr/local/Cellar/node/10.3.0/bin/node" "/usr/local/bin/npm" "run" "postinstall"
8 verbose node v10.3.0
9 verbose npm  v6.1.0
10 error missing script: postinstall
11 verbose exit [ 1, true ]

**Comment by fallenturtle** [comment]
seems the problem was that I needed to install the plugin using this command:
npm run patternlab -- install --plugins plugin-node-scsstojson

**Comment by dave-cross** [comment]
Doh! Lots of new stuff in PL3. I'll have to get in there and play around to see what's breaking and if I can make things better.

Glad you found a solution, @fallenturtle. Thanks for posting it.

**Comment by fallenturtle** [comment]
Really I think I should have investigated more before I posted, which is why I closed it, so I'm sorry for polluting up your issues queue. I disovered later that the issues I was running into with trying to use your plugin I also ran into trying to use the tab plugin, which seems to be the go-to example for all pl node plugins, so the issues are with PL3. Right now it seems with any plugin even if you install it correctly when you try to compile you get an error that says, "Error: ENOENT: no such file or directory, scandir 'plugin-'" I suspect I might need to figure out how to use Evan's scsstojson outside of a plugin wrapper.
